internal/pkg/template/template_test.go

```diff
@@ -34,7 +34,7 @@ func TestTemplate_ReplaceKeys(t *testing.T) {
 			New: model.ConfigRowKey{
 				BranchId:    1,
 				ComponentId: `foo.bar`,
-				ConfigId:    `12`,
+				ConfigId:    `config-in-template`,
```

Comment:
Tu bola chyba, a kedze sa tento cely `ConfigRowKey` nenahradzoval, tak to preslo,
Nahradilo sa iba samostatne `ConfigId` a `Id`.